package-lock.json

```diff
@@ -16,7 +16,7 @@
       "integrity": "sha1-wHshHHyVLsH479Uad+8NHTmQopI=",
       "dev": true,
       "requires": {
-        "balanced-match": "1.0.0",
+        "balanced-match": "^1.0.0",
```

Comment:
The package-lock has a whole lot of updates to not lock to a specific version.  Not that it matters a lot, since it's all dev dependencies, but do you know why that happened?  I haven't seen this before so I'm curious.

Reply:
I think it makes sense to just leave this out, or I can do that when merged to master

Reply:
errrr pass :) npm 6 + my particular settings side affect i think, ill take it out ....